### ISSUE TITLE
Nose Gear improvement

### ISSUE THREAD

**Issue by KBNOtaku** [magknight/787aviatorsTracker#553]
**Describe the bug**
Nose gear wheel texture is much lighter then the main gear.
Nose Gear isnt touching the Ground.

**To Reproduce**
Lokk at the Nosegear at daytime on an non sloped runways etc...

**Expected behavior**
Nose gear wheel shoudl have the same color as the main gear.
Nose gear should touch the ground.

**Screenshots**
![X-Plane_2021-04-07_12-37-19](https://user-images.githubusercontent.com/56799893/114139300-28066900-990f-11eb-9351-a371aa1d9d0e.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - X-Plane version [e.g. 11.50r3]
 - Version 1.6.7-alpha17

**Additional context**
The wheel color should be a normal map issue?

**Comment by aeroniemi** [comment]
I think the new nosegear colour/normals look much better than the old, just that they should be replicated on the main gear - opinions?

**Comment by magknight1** [comment]
I agree that main landing gear should look like nose gear. Someone needs to modify the nosewheel in planemaker so it touches the ground.

**Comment by KBNOtaku** [comment]
yeah ok, but it should look the same.

**Comment by magknight1** [comment]
yes, it should look the same.
